bindings/wasm/src/account/types/encryption_options.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use identity::account_storage::EncryptionOptions;
+use serde::Deserialize;
+use serde::Serialize;
+use wasm_bindgen::prelude::*;
+
+use crate::account::types::WasmCEKAlgorithm;
+use crate::account::types::WasmEncryptionAlgorithm;
+use crate::error::Result;
+use crate::error::WasmResult;
+
+/// Object used for defining the algorithms used for encrypting/decrypting data
+#[derive(Clone, Serialize, Deserialize)]
+#[wasm_bindgen(js_name = EncryptionOptions, inspectable)]
+pub struct WasmEncryptionOptions(EncryptionOptions);
+
+#[wasm_bindgen(js_class = EncryptionOptions)]
+impl WasmEncryptionOptions {
+  /// Creates an `EncryptionOptions` object.
+  #[wasm_bindgen(js_name = new)]
+  pub fn new(
```

Comment:
Should use `constructor`.

```suggestion
  /// Creates an `EncryptionOptions` object.
  #[wasm_bindgen(constructor)]
  pub fn new(
```